tests/import_test.py

```diff
@@ -2,22 +2,395 @@
+                    "b = ¤call_2",
+                    "save_3_a = a",
+                    "save_3_b = b",
+                    "temp_3_s = 'tasting'",
+                    "s = temp_3_s",
+                    "Function Entry C.foo",
+                    "ret_per_se.foo = s + 'see'",
+                    "Exit C.foo",
+                    "a = save_3_a",
+                    "b = save_3_b",
+                    "¤call_3 = ret_per_se.foo",
+                    "c = ¤call_3",
+                    "save_4_a = a",
+                    "save_4_b = b",
+                    "save_4_c = c",
+                    "temp_4_s = 'peking'",
+                    "s = temp_4_s",
+                    "Function Entry D.foo",
+                    "ret_duck_house.foo = s + 'dee'",
+                    "Exit D.foo",
+                    "a = save_4_a",
+                    "b = save_4_b",
+                    "c = save_4_c",
+                    "¤call_4 = ret_duck_house.foo",
+                    "d = ¤call_4",
+                    "Exit module"]
+
+        for node, expected_label in zip(self.cfg.nodes, EXPECTED):
+            self.assertEqual(node.label, expected_label)
+
+    def test_multiple_functions_with_aliases(self):
+        file_path = os.path.normpath('example/import_test_project/multiple_functions_with_aliases.py')
+        project_path = os.path.normpath('example/import_test_project')
+
+        project_modules = get_modules_and_packages(project_path)
+        local_modules = get_directory_modules(project_path)
+
+        self.cfg_create_from_file(file_path, project_modules, local_modules)
+
+        EXPECTED = ["Entry module",
+                    "Module Entry A",
+                    "Module Exit A",
+                    "temp_1_s = 'mutton'",
+                    "s = temp_1_s",
+                    "Function Entry B",
+                    "ret_keens = s",
+                    "Exit B",
+                    "¤call_1 = ret_keens",
+                    "a = ¤call_1",
+                    "save_2_a = a",
+                    "temp_2_s = 'tasting'",
+                    "s = temp_2_s",
+                    "Function Entry C",
+                    "ret_C = s + 'see'",
+                    "Exit C",
+                    "a = save_2_a",
+                    "¤call_2 = ret_C",
+                    "b = ¤call_2",
+                    "save_3_a = a",
+                    "save_3_b = b",
+                    "temp_3_s = 'peking'",
+                    "s = temp_3_s",
+                    "Function Entry D",
+                    "ret_duck_house = s + 'dee'",
+                    "Exit D",
+                    "a = save_3_a",
+                    "b = save_3_b",
+                    "¤call_3 = ret_duck_house",
+                    "c = ¤call_3",
+                    "Exit module"]
+
+
+        for node, expected_label in zip(self.cfg.nodes, EXPECTED):
+            self.assertEqual(node.label, expected_label)
+
+    # def test_init(self):
```

Comment:
As I started to try to get this to pass I thought the PR would be too big so I just pushed and wrote

```python
raise Exception("TODO: Handle __init__ files")
```